reccmp/isledecomp/compare/queries.py

```diff
@@ -0,0 +1,38 @@
+from typing import Iterable, NamedTuple
+from reccmp.isledecomp.types import EntityType
+from .db import EntityDb
+
+
+class OverloadedFunctionEntity(NamedTuple):
+    orig_addr: int | None
+    recomp_addr: int | None
+    name: str
+    symbol: str | None
+    # The number for this repeated name, starting at 1, ordered by orig_addr (nulls last), then recomp_addr.
+    sequence: int
```

Comment:
Maybe `index` instead of `sequence`? If a variable is named `sequence`, I'd expect it to be iterable, like a list

Reply:
`number` is fine with me, other options could be `nth`, `repeated_index`, `repeated_name_index`, or something along these lines